scripts/ci/lib.sh

```diff
@@ -502,6 +502,7 @@ mark_collector_release() {
     export CIRCLE_PULL_REQUEST="https://prow.ci.openshift.org/view/gs/origin-ci-test/logs/${JOB_NAME}/${BUILD_ID}"
     export GITHUB_TOKEN_FOR_PRS="${GITHUB_TOKEN}"
     /scripts/create_update_pr.sh "${branch_name}" collector "Update RELEASED_VERSIONS" "Add entry into the RELEASED_VERSIONS file"
```

Comment:
How is this supposed to work? Is it meant to be invoked in a container or something?

Reply:
this was supposed to be executed in the CircleCI job. The script assumes that some env variables are given - which is the case only in CircleCI. Here it was patched by defining the vars that it requires.

Reply:
I meant the fact that it assumes a `scripts` directory right under the filesystem root.